charts/kamaji-etcd/templates/etcd_sts.yaml

```diff
@@ -48,8 +57,17 @@ spec:
           volumeMounts:
             - name: data
               mountPath: /var/run/etcd
+            {{- if .Values.certManager.enabled }}
+            - name: server-cert
+              mountPath: /etc/etcd/tls/server
+              readOnly: true
+            - name: peer-cert
+              mountPath: /etc/etcd/tls/peer
+              readOnly: true
+            {{- else }}
```

Comment:
Couldn't we use the project volumes to avoid the `if/else` conditions in the `command` section?

https://github.com/clastix/kamaji-etcd/blob/a7af8f76b8ff43d572454ae16f2a816b4f3e3681/charts/kamaji-etcd/templates/etcd_sts.yaml#L85-L99